.ruby-version

```diff
@@ -1 +1 @@
-ruby-3.2.2
+ruby-3.3.0
```

Comment:
`rbenv` seems to fail to automatically detect the version with this prefix.

```suggestion
3.3.0
```

Running `rbenv install` raised the following error:

```
❯ rbenv install
ruby-build: definition not found: ruby-3.3.0

See all available versions with `rbenv install --list-all'.

If the version you need is missing, try upgrading ruby-build:

  brew upgrade ruby-build
```